ehitchhiking-ui/src/styles/modals.sass

```diff
@@ -1,2 +1,52 @@
+@import 'fonts-constants'
+@import 'colors'
+@import 'constants'
+@import 'mixins'
+
+.modal-wrapper
+  overflow: hidden
+  position: relative
+  .modal-header
+    width: 100%
```

Comment:
there is no styles for modal title. 
For example
.modal-title {
text-align: center;
text-transform: capitalize;
}

Reply:
When we use the <span> tag, default styles are appllied and the .modal-title class is not necessary

Reply:
text-transform: capitalize; is not set by default